content/manager/upload-blueprint.md

```diff
@@ -6,43 +6,42 @@ draft: false
 weight: 400
 ---
 
-Before Cloudify Manager can deploy your blueprint, you must upload it. You can upload a blueprint using the CLI or (for Premium users) the Cloudify Web interface.
+Before you can deploy a blueprint, you must upload the blueprint to the Cloudify Manager. You can upload a blueprint using the CLI. Premium users can also upload using the Cloudiy Web Interface.
```

Comment:
Web interface is now available to community edition as well. Let's just say - you can upload a blueprint using either the Cloudify CLI or the web console.

Reply:
This is for 4.1. Does that apply to 4.1 also?

Reply:
No, you are right.